test/ice_nucleation_calibration.jl

```diff
@@ -0,0 +1,278 @@
+import EnsembleKalmanProcesses as EKP
+import EnsembleKalmanProcesses.ParameterDistributions
+import Random
+import Distributions
+import CairoMakie as MK
+import LinearAlgebra
+import OrdinaryDiffEq as ODE
+
+import ClimaParams as CP
+import CloudMicrophysics as CM
+import CloudMicrophysics.Parameters as CMP
+import Thermodynamics as TD
+
+import Test as TT
+
+# definition of the ODE problem for parcel model
+include(joinpath(pkgdir(CM), "parcel", "Parcel.jl"))
+
+# Define model which wraps around parcel and overwrites calibrated parameters
+function run_model(p, coefficients, IN_mode, FT, IC)
+    # grabbing parameters
```

Comment:
Is the calibration setup you are using in the tests the same as in the papers examples? This seems like a lot of code duplication. 

Would it make more sense to define the calibration once somewhere. And then use it in tests and also use the output of it to generate plots?